docs/_posts/2017-08-24-ChenAtARL.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: post
+title: Prof. Li give talks about Cloudberry at Huawei, Google, Teradata, and ARL
```

Comment:
Please fix the grammatical error in this sentence.

Reply:
done.